keystore/src/entities/platform/generic/proteus/prekey.rs

```diff
@@ -131,18 +131,14 @@ impl EntityBase for ProteusPrekey {
 
     async fn delete(
         conn: &mut Self::ConnectionType,
-        ids: &[crate::entities::StringEntityId],
+        ids: crate::entities::StringEntityId<'_>,
```

Comment:
this should be id if it's just one

Reply:
fixed by https://github.com/wireapp/core-crypto/pull/695/commits/7beae609399c01513b60bc51267a1be369c2048e